mlflow_elasticsearchstore/elasticsearch_store.py

```diff
@@ -92,7 +93,9 @@ def _hit_to_mlflow_run_data(self, hit: Any, columns_to_whitelist_key_dict: dict)
         return RunData(metrics=metrics, params=params, tags=tags)
 
     def _hit_to_mlflow_metric(self, hit: Any) -> Metric:
-        return Metric(key=hit.key, value=hit.value if not hit.is_nan else float("nan"),
+        return Metric(key=hit.key,
+                      value=hit.value if not (hasattr(hit, 'is_nan')
```

Comment:
Why do you need to check with hasattr ? Aren't you supposed to always have is_nan since it is in your schema ?

Reply:
some deeper run do not have it for some reasons and the UI crashes because of it

Reply:
I can either put a default value for is_nan when I make the dump if it is not present or leave it that way

Reply:
let's leave it that way it's fine